.github/actions/setup_python/action.yml

```diff
@@ -19,30 +19,35 @@ inputs:
     description: 'If the action should show the share space occupied by cache'
     required: false
     default: 'false'
+  skip-python-install:
+    description: 'Whether to skip Python installation'
+    required: false
+    default: 'false'
```

Comment:
do we need to call this action at all when we work with our custom docker images?